back-end/hub-core/src/main/java/io/apicurio/hub/core/integrations/http/HttpEventSink.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.hub.core.integrations.http;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ByteArrayEntity;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.apicurio.hub.core.integrations.ApicurioEventType;
+import io.apicurio.hub.core.integrations.EventSink;
+
+@ApplicationScoped
+public class HttpEventSink implements EventSink {
+
+    private static final Logger log = LoggerFactory.getLogger(HttpEventSink.class);
+
+    private HttpClient httpClient;
+
+    @Inject
+    HttpSinksConfiguration configuration;
+
+    @Override
+    public String name() {
+        return "http sink";
+    }
+
+    @Override
+    public boolean isConfigured() {
+        return configuration.isConfigured();
+    }
+
+    @Override
+    public void handle(ApicurioEventType type, String designId, byte[] data) {
```

Comment:
The API design id is not used?

Reply:
Maybe I've changed my mind about the interface method. :)

Reply:
hehe I forgot about this :), I'll remove that argument